lib/cc/engine/analyzers/php/main.rb

```diff
@@ -7,6 +7,7 @@ module Engine
     module Analyzers
       module Php
         class Main
+          DEFAULT_PATHS = ["**/*.php"]
```

Comment:
PHP also uses `.inc` and `.module` files - we support these on the Classic analysis